gateway-service-knoxsso/pom.xml

```diff
@@ -81,5 +91,12 @@
             <artifactId>gateway-test-utils</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.5</version>
```

Comment:
See other comment about removing gson altogether if possible. Otherwise don't need version and scope on this.

Reply:
Done